images/macos/provision/core/vsmac.sh

```diff
@@ -4,14 +4,18 @@ source ~/utils/xamarin-utils.sh
 
 install_vsmac() {
     local VSMAC_VERSION=$1
-    if [ $VSMAC_VERSION == "latest" ]; then
+    local VSMAC_DEFAULT=$2
+    if [ $VSMAC_VERSION == "2019" ]; then
         VSMAC_DOWNLOAD_URL=$(curl -sL "https://aka.ms/manifest/stable" | jq -r '.items[] | select(.genericName=="VisualStudioMac").url')
+    elif [ $VSMAC_VERSION == "2022" ]; then
+        VSMAC_DOWNLOAD_URL=$(curl -sL "https://aka.ms/manifest/stable-2022" | jq -r '.items[] | select(.genericName=="VisualStudioMac").url')
     elif [ $VSMAC_VERSION == "preview" ]; then
         VSMAC_DOWNLOAD_URL=$(curl -sL "https://aka.ms/manifest/preview" | jq -r '.items[] | select(.genericName=="VisualStudioMac").url')
     else
         VSMAC_DOWNLOAD_URL=$(buildVSMacDownloadUrl $VSMAC_VERSION)
```

Comment:
I wonder if we still need this as it looks like all the downloads are now hosted in https://download.visualstudio.microsoft.com, not in dl.xamarin.com

Reply:
It still works - https://dl.xamarin.com/VsMac/VisualStudioForMac-17.0.0.8989-x64.dmg